{"name": "test_can_create_a_courier", "status": "failed", "statusDetails": {"message": "AssertionError: Status code 201 was expected and code 409 was returned.\nassert 409 == 201\n +  where 409 = <Response [409]>.status_code", "trace": "self = <test.test_CourierCreating.TestCreateСapability object at 0x00000228F2CE1A50>, create_courier = <Response [409]>\n\n    def test_can_create_a_courier(self, create_courier):\n>         assert create_courier.status_code == 201, f\"Status code 201 was expected and code {create_courier.status_code} was returned.\"\nE         AssertionError: Status code 201 was expected and code 409 was returned.\nE         assert 409 == 201\nE          +  where 409 = <Response [409]>.status_code\n\ntest\\test_CourierCreating.py:6: AssertionError"}, "start": 1692635383773, "stop": 1692635383773, "uuid": "154fa4a9-ef53-41fb-bb64-f656576e0a67", "historyId": "606e53e34df54a6826c195407052f41d", "testCaseId": "606e53e34df54a6826c195407052f41d", "fullName": "test.test_CourierCreating.TestCreateСapability#test_can_create_a_courier", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_CourierCreating"}, {"name": "subSuite", "value": "TestCreateСapability"}, {"name": "host", "value": "DESKTOP-36NVPK8"}, {"name": "thread", "value": "8420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_CourierCreating"}]}